### DISCUSSION TITLE
web webman-mod problems with ps2

### DISCUSSION THREAD

**Discussion by e1milio** [aldostools/webMAN-MOD#1011] [General]
I have a ps3 fat with custom firmware evilnat 4.90 pex and webman mod updated to its latest version.
I like to monitor the temperature with http://ps3ip/tempc.html from my PC while I'm running a game, but the page stops working when I'm running a PS2 game. Does this have a solution?
Another problem I have is that the minimum and maximum fan speed settings do not work when I play a PS2 game, it always takes the manual setting for PS2, how can I get it to take the automatic setting?

**Comment by aldostools** [comment]
Unlike other emulated systems on PS3, the PS2 emulation requires to unload LV2 kernel (GameOS) to load the emulator.
In the process, all VSH plugins and processes running in background are unloaded, including webMAN MOD.

The reason of this behavior is that PS2 emulation requires all available memory and CPU capacity. This is the way PS2 emulation works on PS3 and cannot be changed.

As VSH plugins cannot work on PS2 mode, webMAN MOD has a special fan mode for PS2 emulation in /setup.ps3 where you have to set a fixed fan speed. As PS2 emulation causes much more heat than other games, the recommended setting is 40% or higher if you get an average temperature over 75°C (167°F).

Evilnat 4.90 PEX includes a modified PS2 emulator (ps2_netemu) that shows the temperature when you press PS button.

**Reply by e1milio** [reply]
thank you
